lib/c0/c0_kvmultiset.c

```diff
@@ -69,7 +69,7 @@ struct c0_kvmultiset_impl {
     atomic_ulong          c0ms_txhorizon;
     size_t                c0ms_used;
     atomic_ulong         *c0ms_kvdb_seq;
-    void                **c0ms_stashp;
+    void * _Atomic       *c0ms_stashp;
```

Comment:
Didn't even know you could put _Atomic in this location.

Reply:
Yeah, took me a while to figure out.  I tried to use atomic_inptr_t to avoid the hard work, but I knew you can make pretty much anything atomic with _Atomic (even large structures), and the compiler will use locks or whatever it needs to do to make it atomic.